src/core_loggers/file_logger.ts

```diff
@@ -45,7 +45,10 @@ export class FileLogger extends Logger {
    * @return string|void
    *     Returns the log message which is used for unit testing purposes.
    */
-  public write(logMethodLevelDefinition: any, message: string): string | void {
+  public write(
+    logMethodLevelDefinition: Drash.Interfaces.LogLevelStructure,
+    message: string,
+  ): string | void {
```

Comment:
To also include something like `and returns void if blah blah blah`?

Reply:
47c55d3